src/validation.cpp

```diff
@@ -3511,9 +3538,9 @@ CBlockIndex * InsertBlockIndex(uint256 hash)
     return pindexNew;
 }
 
-bool static LoadBlockIndexDB(const CChainParams& chainparams)
+bool CChainState::Load(CBlockTreeDB& blocktree)
```

Comment:
What lambda?
Anyway, if we plan not to do #9176 and moving the check instead, then there's no point in passing Consensus::Params to CChainState::Load if it's going to be only temporary.